.travis.yml

```diff
@@ -1,3 +1,19 @@
 language: java
 dist: trusty
 jdk: oraclejdk8
+secure: "H5axvvBaHiHfeCCER0VAB+KM+9frKr5cBAtqy+QpIlsSp7oN4wayxxrfWM0ll1dw+Gv7Uzr7rgRQDWcM/ELf+7c9Wi9ug+jZjYcE5t9mYrB9m8GHnac7S4Hlsx29qDXVzAnYqZ3vIChCCr4odf76SEYqPFQ/KOCLA5Cd6aGrKoYn7jqBh3tFpIQ70TJvzbNR4tYBPeytcwVEioEegFGhxRpIzLqRCxzdwoaP3qO2/OO1BGWkzIKIiYpM6tuRTIH8n4SoPJ/pEXVkEdnf+NjLJQ6MvihcHYl+mD/O5VsqJp87CZlUVW8WHgW9tWAauTsTn8evV26kXCx/s940RYIddOj5T8QiLNCuZWtmnBIFdPGCVcI22SFEvfFEW7Bm1IjofV6dHVhNkjuilBFj7v3XPDXEAm99OEMckNftuEDsDVBctnDpA1b718nA1HB0u6Mge+5+NiV438/Xnge6aLu4+BQs0kLlHMyiQUvDYAsbn4AnK1tM0ICZ+ZnInPbi3ezIbjdfI8PgUUySOqcBjL7otlsBhZ7o1uDmZGiz12xEZCh0EVZD61ys1A/uVIrTGnSM8YUsRMOS/mNwYhe0opAB92F6d1aVo/tQsE/7H6Zkh3clC1xLdq/txO8z3CzVqXvmqYkCOX2amture5Ur3z11jqbXAetnEeGCJgJNvMgzcPI="
+
+before_deploy:
+  - ./gradlew clean
+  - ./gradlew jar
```

Comment:
should this build a jar including all depended third part libs, like pcap4j, gson, etc?
if not, then the users of this trex-java-sdk have to explicitly introduce those third part libs as dependencies of their repos.

Reply:
yes and no. It dont mean they have to introduce them explicity, it only mean it will get downloaded when building with this dependency. Maybe not all want this behaviour, we could generate both type of jars